tests.nix

```diff
@@ -98,13 +98,23 @@ let crate2nix = pkgs.callPackage ./default.nix {};
          }
 
          {
-             name = "sample_project_cfg_test";
-             src = ./sample_projects/cfg-test;
-             cargoToml = "Cargo.toml";
-             expectedOutput = "Hello, cfg-test!";
+            name = "sample_project_cfg_test";
+            src = ./sample_projects/cfg-test;
+            cargoToml = "Cargo.toml";
+            expectedOutput = "Hello, cfg-test!";
+            pregeneratedBuild = "sample_projects/cfg-test/Cargo.nix";
+          }
+
+         {
+            name = "sample_project_cfg_test-with-tests";
+            src = ./sample_projects/cfg-test;
+            cargoToml = "Cargo.toml";
+            expectedOutput = "Hello, cfg-test!";
```

Comment:
Can you change cfg-test in such a way that you actually notice the difference? E.g. use annotations to print something else?

Reply:
Yeah, I could probably use some kind of 

```
#[cfg(test)]
println!("foo");
```